lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -54,6 +55,12 @@ def image_alt_text(label)
                 label
               end
             end
+
+            def product_tags(input)
+              input['tags'].each_with_object([]) do |tag, array|
```

Comment:
Is there a reason we can't just do `input['tags'].map { |tag| tag['name'] }.join(', ')`?

Reply:
you also could do something fancy with inject/reduce, but doing a map then a join is prob the most readable approach

Reply:
I used ```each_with_object``` for no particular reason, it was just on the top of my head and ```map``` didn't come to my mind. You're absolutely right and ```map``` seems to be more readable than ```each_with_object```

Reply:
👍 to Jordan's recommendation.